Source/WebCore/platform/graphics/ColorTypes.h

```diff
@@ -721,6 +736,26 @@ struct RGBA {
     {
     }
 
+    uint8_t red() const
+    {
+        return (value >> 24) & 0xFF;
+    }
+
+    uint8_t green() const
+    {
+        return (value >> 16) & 0xFF;
+    }
+
+    uint8_t blue() const
+    {
+        return (value >> 8) & 0xFF;
+    }
+
+    uint8_t alpha() const
+    {
+        return value & 0xFF;
+    }
+
     uint32_t value;
```

Comment:
Let's just serialize this uint32_t instead of breaking it into pieces.  Then this will need no modification.